web/sites/default/config/views.view.content.yml

```diff
@@ -727,24 +727,19 @@ display:
       style:
         type: table
         options:
-          grouping:
-            -
-              field: field_migration_date
-              rendered: true
-              rendered_strip: false
+          grouping: {  }
```

Comment:
@RichardDavies Not sure what's causing the config change on this view. I noticed that the table grouping on the multi-dev looks different from Live:
https://pe-479-employees.pantheonsite.io/admin/content/contacts/multnomah
https://employees.portland.gov/admin/content/contacts/multnomah

Reply:
Thanks for catching this! This was indeed an unintentional change. Only the default sort order on /admin/content should change. All of the view displays were originally using the same table settings, which were set to display grouped by migration date and sorted by migration date.

But since /admin/content doesn't have a migration date field there was no default sort on that display. I've now set the two contact displays to use their own unique settings so they can be sorted by migration date, and /admin/content is using the default sort by updated date.

Reply:
Yes, that "Change this for all display" option is easy to accidentally set. I've done it many times. :)